src/github.com/getlantern/flashlight/ui/ui.go

```diff
@@ -0,0 +1,72 @@
+package ui
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"path"
+	"path/filepath"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/tarfs"
+	"github.com/skratchdot/open-golang/open"
+)
+
+const (
+	localResourcesPath = "../ui/app"
```

Comment:
I've been running flashlight from the flashlight-build directory. So this is no longer looking for a ui directory in ```src/github/getlantern```? Running flashlight from ```src/gitgethub.com/getlantern/flashlight``` I'm getting ```go run flashlight.go -role client -addr localhost:10080 -uiaddr localhost:16785 -cloudconfig http://s3.amazonaws.com/lantern_config/cloud.valencia.yaml.gz```
```# command-line-arguments
./flashlight.go:228: undefined: Asset```